README.md

```diff
@@ -1 +1,76 @@
-# Capstone-Project---University-Campus-Hour
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+#### University Campus Hour website
+Built as Microverse first capstone project
+
+### Project Description
+
+In this simple University Campus Hour project, I have tried to imitate a design as much as possible adding some designing portions from my ideas. All the designs are done using HTML and CSS. Some functionalities in the project have been made by JavaScript.
+
+*You can find the original design [here](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).*
+
+## Built With
+
+- HTML, CSS, and JavaScript
+
+## Getting Started
+To clone the repository in local environment try following steps.
+
+### Prerequisites
+
+- [x] A web browser like Google Chrome.
+- [x] A code editor like Visual Studio Code with Git and Node.js.
+
+You can check if Git is installed by running the following command in the terminal.
+```
+$ git --version
+```
+
+Likewise for Node.js and npm for package installation.
+```
+$ node --version && npm --version
+```
+### Setup
+
+Clone the repository using [this link](https://github.com/PrangonGhose/My-Portfolio.git)
+
+### Install
+
+In the terminal, go to your file directory and run this command.
+
+```
+$ git clone git@github.com:mavericks-db/portfolio.git
+```
+
+### Usage
+
+This website is applicable for both mobile and desktop version.
+
+### Run tests
+
+## Authors
+
+👤 **Prangon Ghose**
+
+- GitHub: [@PrangonGhose](https://github.com/PrangonGhose)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome! Add suggestions by opening new issues.
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+- Original designer [Cindy Shin](https://www.behance.net/adagio07)
+- The [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/)
+- [Microverse](https://www.microverse.org/)
+- Code Reviewers
+- Coding Partners
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Kindly I recommend you to update your new [README template](https://github.com/microverseinc/readme-template) and All main headers from the README template should be present in a yours repo. But customize according to your project and please Make sure that you added the section "Future Features", including 1-3 potential features to be added.